README.md

```diff
@@ -31,6 +32,10 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## About
+
+This is a repository created and maintained by the IBM Watson Discovery UI team. It contains a collection of components, to be used by internal and external applications to query [Watson Discovery](https://www.ibm.com/products/watson-discovery) projects.
```

Comment:
Add a line pointing to the storybook. Something like "Take a look at our [Storybook](...) for info on these components."